### DISCUSSION TITLE
Is "CyberCodeOnline/contribution/words.json" currently translatable?

### DISCUSSION THREAD

**Discussion by johnruby** [DexterHuang/CyberCodeOnline#1389] [General]
As the title, is [CyberCodeOnline/contribution/words.json](https://github.com/DexterHuang/CyberCodeOnline/blob/master/contribution/words.json), which includes the relevant text for dungeon quest event, currently translatable?

If the file is open to translation, I'm willing to start translating the file into Chinese and adding original content, but I'm not sure which directory I should put the translated file in.

**Comment by DexterHuang** [comment]
a part of that file will be translatable very soon, in the next update :)

**Reply by johnruby** [reply]
Got it, many thanks!

**Reply by ejNatt** [reply]
Your welcome

**Reply by Want2BeRocker** [reply]
Hello Dexter, I wanted to translate game to uzbek and don't know how (there's no any uz-uz.json files). So how can I do it?
